### DISCUSSION TITLE
Attenuator DIY questions

### DISCUSSION THREAD

**Discussion by bkill504** [gpstar81/GPStar-proton-pack#247] [General]
I have some questions about some of the components I'm hoping you can answer. First, I have the new Frutto tech bar graph with the black board, version 3.0 I guess you could say. Do I still need to run resistors to it on the breakout board or are those built in now? Second, I'm having trouble finding the 3.8k resistors, would I be able to use 3.9K or would that be to much? Thanks for your help in advance.

**Comment by bkill504** [comment]
The resistors for the bar graph can be removed if it's the 3.0 version. Now I need to figure out what I did wrong with my toggle switches.

**Comment by DustinGrau** [comment]
Sorry, I didn't check this board in a while. That's correct, with the 3.0 bargraph you no longer need the pull-up resistors on the SDA/SCL pins.
